src/pages/sectioning/Confirm.tsx

```diff
@@ -91,6 +80,17 @@ function Confirm() {
           </div>
         </div>
       </div>
+
+      <OperationCompleteModal
+        message={"Sectioning Saved"}
+        show={model.isDone}
+        onReset={reload}
+      >
+        <p>
+          If you wish to start the process again, click the "Reset Form" button,
+          otherwise you can return to the Home screen.
```

Comment:
```suggestion
          If you wish to start the process again, click the "Reset Form" button.
          Otherwise you can return to the Home screen.
```